src/pykiso/exceptions.py

```diff
@@ -74,3 +74,14 @@ def __init__(self, name: str) -> None:
             "needs to start with a letter or underscore"
         )
         super().__init__(self.message)
+
+
+class NoTestsFoundException(Exception):
```

Comment:
I would rather go for the already existing TestCollectionError instead of creating specific exceptions for each possible use case, the TestCollectionError could be made more generic to fit this use case too